scripts/blog.js

```diff
@@ -1,2 +1,277 @@
+        author:      'Horse Whisperer',
+        authorUrl:   'http://www.imdb.com/title/tt0119314/',
+        publishedOn: '2015-10-17',
+        body:        '<p>And practical explanations. And practical explanations. And practical explanations. And practical explanations. This is a very special technique that I have never seen. And practical explanations. This is a very special technique that I have never seen. And practical explanations. In less than three weeks you can be looking at things differently, more creatively, if you. And practical explanations.</p><p><img src="http://lorempixel.com/400/200/animals/" class="pull-left">To discourage you by telling. -And Gain Power By Learning Ways To Become Peaceful. -And Gain Power By Learning Ways To Become Peaceful. The real goal of any talk or speech. To discourage you by telling. -And Gain Power By Learning Ways To Become Peaceful. The real goal of any talk or speech. To discourage you by telling. The real goal of any talk or speech. -And Gain Power By Learning Ways To Become Peaceful. -And Gain Power By Learning Ways To Become Peaceful. <img src="http://lorempixel.com/400/300/animals/" class="pull-right">The real goal of any talk or speech.</p><p>-And Gain Power By Learning Ways To Become Peaceful. Don\'t stubbornly. To discourage you by telling. To discourage you by telling. Don\'t stubbornly. Don\'t stubbornly. Space in your house How to sell faster than your neighbors How to make a strategic use. <img src="http://lorempixel.com/400/200/animals/" class="pull-left">To discourage you by telling. To discourage you by telling. Don\'t stubbornly. Don\'t stubbornly. -And Gain Power By Learning Ways To Become Peaceful. Don\'t stubbornly. Don\'t stubbornly. Don\'t stubbornly. Don\'t stubbornly. Don\'t stubbornly. To discourage you by telling. Don\'t stubbornly. Don\'t stubbornly.</p><p>-And Gain Power By Learning Ways To Become Peaceful. I m going to outline 14 different ways that I ve found you. To discourage you by telling. To discourage you by telling. To discourage you by telling. Space in your house How to sell faster than your neighbors How to make a strategic use. To discourage you by telling. To discourage you by telling. To discourage you by telling. -And Gain Power By Learning Ways To Become Peaceful. The real goal of any talk or speech. To discourage you by telling.</p><p>Then I decided to try an experiment and put all of my hard work to the test. The next morning I got up and walked straight to my computer.. And practical explanations. And practical explanations. And practical explanations. Why many people achieve very little despite spending most of their time. This is a very special technique that I have never seen. And practical explanations. Why many people achieve very little despite spending most of their time. And practical explanations.</p>'
+
+      },
+    ];
+
+};
+
+// CompareByDate function for sorting the articles
+// Taken from Microsoft website
```

Comment:
Good job showing attribution. Try to be more specific if possible, like a URL to where exactly you found this code.